en_US/data-integration/data-bridge-iotdb.md

```diff
@@ -140,23 +114,48 @@ You can continue to create a rule to forward data to the new Apache IoTDB bridge
      "root/#"
    ```
 
-5. Click the **Add Action** button, select **Forwarding with Data Bridge** from the dropdown list, and then select the data bridge you just created under **Data Bridge**.
+5. Click the **Add Action** button, and select **Forwarding with Data Bridge** from the dropdown list. With this action, EMQX sends the data processed by the rule to the data bridge.
 
-   :::tip
+6. Click the **+** icon next to the **Data bridge** drop-down box to create a data bridge.
 
-   You can skip this step and the next step if you chose **Create Rule** when creating the bridge itself, it will be pre-configured in that case.
+7. Select **Apache IoTDB** from the **Type of Data Bridge** drop-down list. Fill in the required fields (marked with an asterisk).
 
-   :::
+8. Enter a name for the data bridge. The name should be a combination of upper/lower case letters and numbers.
+
+9. Enter the connection information:
 
-6. Click the **Add** button to finish the setup.
+   * **IoTDB REST Service Base URL**: Enter `http://localhost:18080`, or the actual hostname/IP if the IoTDB server is running remotely.
 
-7. Click the **Create** button at the bottom of the page to finish the setup.
+   * **IoTDB Version**: Select the version of the IoTDB server you have installed.
 
-Now a rule to forward data to Apache IoTDB via the data bridge is created. You can click **Integration** -> **Flows** to view the topology. It can be seen that the messages under the topic `root/#` are sent and saved to Apache IoTDB.
+   * **Username**: Enter the IoTDB username; The default value is `root`.
 
-### Test the Data Bridge and Rule
+   * **Password**: Enter the IoTDB password; The default value is `root`.
+
+   * **Device ID** (optional): Enter a fixed device id used as the name of the device from which the timeseries data is forwarded and inserted into the IoTDB instance.
+
+     :::tip
+
+     When left empty, the device id can also be specified in the publishing message, configured in the rule, or extracted from the topic to which these messages are being sent by converting '/' into '.'. For example, publishing a message to `root/sg27` will result in sending a device name of `root.sg27`. However, the fixed device id configured in this field takes precedence over any methods mentioned previously.
```

Comment:
@sstrigler may correct me, but I can't see this extraction from topic in the current code.

```suggestion
     When left empty, the device id can also be specified in the publishing message or configured in the rule. For example, when publishing a message JSON-encoded with a `device_id` field in it, the value of that field will define the output device id. To define it using the rule engine, one must use a SQL like `select json_decode(payload) as payload, 'my_device' as payload.device_id`. However, the fixed device id configured in this field takes precedence over any methods mentioned previously.
```